src/main/resources/bidder-config/brightroll.yaml

```diff
@@ -44,11 +44,95 @@ adapters:
           - thebeverlyhillsmdsolution.com
           - thecrux.com
           - thehornnews.com
+          - ezmob.com
```

Comment:
Sorry for the late response. 
We found that DSP doesn't support more that 50 badv.

Reply:
Sorry, this is my mistake. The Rubicon exchange has a limit of 50 badv entries. Brightroll does not. This PR is ok.